src/components/Header/index.tsx

```diff
@@ -280,10 +280,10 @@ function Header() {
           <Amount zero={!!userNativeCurrencyBalance?.equalTo('0')}>
             {!account ? (
               '0.000'
-            ) : userNativeCurrencyBalance ? (
-              userNativeCurrencyBalance?.toFixed(3)
-            ) : (
+            ) : !userNativeCurrencyBalance ? (
               <Skeleton width="40px" />
+            ) : (
+              userNativeCurrencyBalance?.toFixed(3)
```

Comment:
Optional chaining is not required here. if `userNativeCurrencyBalance` is undefined, the `Skeleton` component is shown.

Reply:
Fixed!